rust_src/remacs-macros/lib.rs

```diff
@@ -137,7 +139,11 @@ struct CByteLiteral<'a>(&'a str);
 
 impl<'a> quote::ToTokens for CByteLiteral<'a> {
     fn to_tokens(&self, tokens: &mut quote::Tokens) {
-        tokens.append(&format!(r#"b"{}\0""#, self.0));
+        let re = Regex::new(r#"["\\]"#).unwrap();
```

Comment:
The regex should be compiled in a lazy_static! so we do not pay for it in every function call. There are examples in the regex doc https://doc.rust-lang.org/regex/regex/index.html#example-avoid-compiling-the-same-regex-in-a-loop